documentation/docs/user_guide/inputs/meteorological_forcing.md

```diff
@@ -0,0 +1,124 @@
+        SWdown:missing_value = 1.e+20f ;
+    int land(land) ;
+        land:compress = "y x" ;
+    float nav_lat(y, x) ;
+        nav_lat:units = "degrees_north" ;
+        nav_lat:valid_min = -90.f ;
+        nav_lat:valid_max = 90.f ;
+        nav_lat:long_name = "Latitude" ;
+    float nav_lon(y, x) ;
+        nav_lon:units = "degrees_east" ;
+        nav_lon:valid_min = -180.f ;
+        nav_lon:valid_max = 180.f ;
+        nav_lon:long_name = "Longitude" ;
+    float time(tstep) ;
+        time:units = "seconds since 1949-01-01 00:00:00" ;
+        time:title = "Time" ;
+        time:long_name = "Time axis" ;
+        time:time_origin = " 1949-JAN-01 00:00:00" ;
+```
+
+Most of the variables will be structured as SWdown is above. The relationship between “land” and “x” and “y” inside the CABLE netcdf driver is:
+
+```fortran
+y = INT((landGrid(j)-1)/xdimsize)
+x = landGrid(j) - y*xdimsize
+y = y + 1
+```
+
+Note that the number of points simulated by offline CABLE is not specified in the code, but determined here using the number of land points found in the met file. If a patch dimension exists, the `patchfrac` variable gives the fraction of each vegetation patch in each land grid point. CABLE counts the number of "active" patches (patches with non-zero `patchfrac`) as the number of points for simulation.
+
+## Variables
+
+### Required variables
+
+| Name     | Description                               | Units                  |
+|----------|-------------------------------------------|------------------------|
+| `time`   | time of each time step (double precision) | \( s \)                |
+| `SWdown` | Surface incident shortwave radiation      | \( W \cdot m^{-2} \)   |
+| `Tair`   | Surface incident shortwave radiation      | \( K \)                |
+| `Qair`   | Near surface specific humidity            | \( kg \cdot kg^{-1} \) |
+| `Rainf`  | Rainfall rate                             | \( mm \cdot s^{-1} \)  |
+| `Wind` or `Wind_E` and `Wind_E` | Scalar wind speed  | \( m \cdot s^{-1} \)   |
+
+### Optional variables
+
+| Name        | Description | Units |
+|-------------|-----------------------------------------------------------------------------------------------------|-----------------------|
+| `LWdown`    | Surface incident longwave radiation (can be synthesised from `Tair`)                                | \( W \cdot m^{-2} \)  |
+| `PSurf`     | Surface air pressure (can be estimated at a fixed value based on Tair and an "`elevation`" variable | \( Pa \)              |
+| `elevation` | Surface elevation (required if `PSurf` is not present)                                                | \( m \)               |
+| `Snowf`     | Snowfall (assumed to be included in `Rainf` if not present)                                           | \( mm \cdot s^{-1} \) |
+| `CO2air`    | CO$_2$ concentration (assumes a fixed value, determined in the `cable.nml` namelist file)           | \( ppm \)             |
```

Comment:
```suggestion
| `CO2air`    | CO$_2$ concentration (assumes a fixed value, determined in the `cable.nml` namelist file)           | \( ppm \)             |
| `patchfrac` | The fraction of each vegetation patch in each land grid point. CABLE counts the number of "active" patches (patches with non-zero `patchfrac`) as the number of points for simulation. | \( - \) |
```
`patchfrac` was missing as an optional variable so I just moved the description from the text to the table.